README.md

```diff
@@ -1,3 +1,166 @@
 # React Vis Network
 
 [![Build Status](https://travis-ci.org/lifeomic/react-vis-network.svg?branch=master)](https://travis-ci.org/lifeomic/react-vis-network)
+
+
+## Overview
+react-vis-network uses a declarative approach to defining [vis.js](https://github.com/almende/vis). networks. This allows developers to work in React using normal statefull approaches like [Redux](https://github.com/reduxjs/redux) and have the underlying network chart manage it's own lifecycle.
```

Comment:
Two teeny typos I noticed:  The extra period after the vis.js link, and the spelling of `stateful`